### PR TITLE
Chore: Rename Victorops to Splunk OnCall

### LINKED ISSUE #47632: Rename VictorOps notifier to Splunk OnCall
**What happened**:

VictorOps has been acquired by Splunk: https://www.splunk.com/en_us/investor-relations/acquisitions/splunk-on-call.html

The notifier should be updated to reflect the new name.

Care should be taken to migrate existing VictorOps notifiers in existing user configurations. Some potential ways around this:
1. Create a migration to update all old-style notifiers to the new one
2. Only change display names, etc, and keep type identifiers the same
3. Have the code support both type identifiers indefinitely

**What you expected to happen**:

I expected there to not be deprecated names in the notifier list.

**How to reproduce it (as minimally and precisely as possible)**:

Open the Contact Point Type drop-down.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.4.x

### PR DESCRIPTION
**Pull request by krishnemd**
**What is this feature?**

This PR will rename VictorOps to Splunk OnCall as per the issue description. 

**Why do we need this feature?**

To have appropriate naming.

**Who is this feature for?**

This feature is for every end user.

**Which issue(s) does this PR fix?**:

Fixes #47632

**Special notes for your reviewer:**
This comment in issue section https://github.com/grafana/grafana/issues/47632#issuecomment-1653263284 needs to be answered for the proper implementation of the rename. It seems that the PR section is getting communication so I have raised a PR with a simple visual level fix of the problem. Upon answering the above given questions on the specific issue section I can add few commits on this PR if we need a complete rename here.

Please check that:
- [] It works as expected from a user's perspective.
- [] If this is a pre-GA feature, it is behind a feature toggle.
- [] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/old-alerting/notifications.md [hunk 1]
```diff
@@ -164,11 +164,11 @@ To set up PagerDuty, all you have to do is to provide an integration key.
 
 > **Note:** Grafana uses the `Events API V2` integration. This can be configured for each service.
 
-### VictorOps
+### Splunk OnCall
```
Comment (grobinson-grafana) [inline docs/sources/old-alerting/notifications.md:167]: Old alerting is deprecated, so please revert the changes in this file.

File: pkg/services/ngalert/notifier/channels_config/available_channels.go [hunk 2]
```diff
@@ -364,15 +364,15 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 		},
 		{
 			Type:        "victorops",
-			Name:        "VictorOps",
-			Description: "Sends notifications to VictorOps",
-			Heading:     "VictorOps settings",
+			Name:        "Splunk OnCall",
```
Comment (grobinson-grafana) [inline pkg/services/ngalert/notifier/channels_config/available_channels.go:367]: Please don't revert the changes here though as `ngalert` is the new Grafana Alerting.

**Review by grobinson-grafana — CHANGES_REQUESTED** [review]
Thanks for your contribution! Just a couple of small changes please.

File: pkg/services/alerting/notifiers/victorops.go [hunk 3]
```diff
@@ -22,22 +22,22 @@ const alertStateRecovery = "RECOVERY"
 func init() {
 	alerting.RegisterNotifier(&alerting.NotifierPlugin{
 		Type:        "victorops",
-		Name:        "VictorOps",
-		Description: "Sends notifications to VictorOps",
-		Heading:     "VictorOps settings",
+		Name:        "Splunk OnCall",
+		Description: "Sends notifications to Splunk OnCall",
+		Heading:     "Splunk OnCall settings",
 		Factory:     NewVictoropsNotifier,
```
Comment (grobinson-grafana) [inline pkg/services/alerting/notifiers/victorops.go:28]: We should also rename the code https://github.com/grafana/alerting/tree/main/receivers/victorops and the functions that are used here.

**Comment by chauchausoup** [comment]
@grobinson-grafana I have closed the PR due to some internal git merge issue from my side. Sorry for that.

What might be an effective migration plan here ? Editing the on going VictorOps or rolling out parallel clone VictorOps and changing the clone accordingly ?

In the mean time as per your suggestion above, I am setting up a new PR as 
https://github.com/grafana/grafana/pull/75786
and also on /alerting as 
https://github.com/grafana/alerting/pull/126

But it seems that /alerting talks with prometheus fro api_keys so I guess we need to untouch those fields and let them be victorOps or should raise a PR in prometheus too ?

All these operations might need many steps so I recommend I will step by step raising unit PRs and tag you on each ?

First I will start with, `.md files -->> UI fields -->> API endpoints -->> alerting repo -->> prometheus repo`

Let me know you thoughts.

**Pull request closed without merging**